### PR TITLE
odo: add recipe

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

File: recipes/odo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/odo/meta.yaml]: It looks like only `python` and `setuptools` are required to build the package.

File: recipes/odo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - odo
+    - odo.backends
+    - odo.backends.tests
+    - odo.tests
+
+about:
+  home: https://github.com/blaze/odo
+  license: BSD-3-Clause
+  summary: 'Data migration in Python'
+  license_family: BSD
```
Comment (synapticarbors) [inline recipes/odo/meta.yaml]: please include `license_file: LICENSE.txt`

File: recipes/odo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{% set name = "odo" %}
+{% set version = "0.5.0" %}
```
Comment (synapticarbors) [inline recipes/odo/meta.yaml]: It looks like there is a `0.5.1` release only on Github, but not on pypi.

File: recipes/odo/meta.yaml (hunk 1 above)
Comment (nehaljwani) [inline recipes/odo/meta.yaml]: I tried that, but the build fails without these. (Note that I didn't add numpy in the build section)

Comment (nehaljwani) [inline recipes/odo/meta.yaml]: Seems like that is fixed in 0.5.1 .. updating PR now.

**Pull request merged**